galasa-ui/src/utils/functions/translations.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+import { TranslationValues } from 'next-intl';
+import { TranslationFn } from '../types/common';
+
+/**
+ * Function to get the translation for a given key, and if not found, return the key itself.
```

Comment:
Would it be better to return the English variant if the German is missing ? And some log failure in the console log or elsewhere ?
And the key only if the English is missing ?